t/apicast-async-reporting.t

```diff
@@ -1,5 +1,5 @@
 use lib 't';
-use TestAPIcast 'no_plan';
+use Test::APIcast 'no_plan';
```

Comment:
This namespace looks a bit strange to me. Is it common in Perl to start testing libraries with `Test::`?
I guess that's it, because we also have `Test::Nginx`.

Reply:
Yep. And there is Test::Simple and Test::More.

Reply:
:+1: